dreamjob_frontend/src/routes/glassdoor.js

```diff
@@ -15,7 +13,7 @@ class glassdoor extends Component {
   }
 
   componentWillMount(){
-    jobStore.on('glassdoor', this.updateDetails.bind(this))
+    jobStore.on('glassdoor', this.updateDetails.bind(this)) // NEED
```

Comment:
amended comment to id which emissions each component must listen to. refactored and removed superfluous listeners.